src/common/Page.ts

```diff
@@ -2438,26 +2439,26 @@ export class Page extends EventEmitter {
    *   If no path is provided, the image won't be saved to the disk.
    *
    * - `type` : <string> Specify screenshot type, can be either `jpeg` or `png`.
-   *   Defaults to 'png'.
+   *   Defaults to 'png'.</string>
    *
    * - `quality` : <number> The quality of the image, between 0-100. Not
-   *   applicable to `png` images.
+   *   applicable to `png` images.</number>
    *
    * - `fullPage` : <boolean> When true, takes a screenshot of the full
-   *   scrollable page. Defaults to `false`
+   *   scrollable page. Defaults to `false`</boolean>
    *
    * - `clip` : <Object> An object which specifies clipping region of the page.
-   *   Should have the following fields:<br/>
-   *  - `x` : <number> x-coordinate of top-left corner of clip area.<br/>
-   *  - `y` :  <number> y-coordinate of top-left corner of clip area.<br/>
-   *  - `width` : <number> width of clipping area.<br/>
-   *  - `height` : <number> height of clipping area.
+   *   Should have the following fields:</Object><br/>
+   *  - `x` : <number> x-coordinate of top-left corner of clip area.</number><br/>
+   *  - `y` :  <number> y-coordinate of top-left corner of clip area.</number><br/>
+   *  - `width` : <number> width of clipping area.</number><br/>
+   *  - `height` : <number> height of clipping area.</number>
    *
    * - `omitBackground` : <boolean> Hides default white background and allows
-   *   capturing screenshots with transparency. Defaults to `false`
+   *   capturing screenshots with transparency. Defaults to `false`</boolean>
    *
    * - `encoding` : <string> The encoding of the image, can be either base64 or
-   *   binary. Defaults to `binary`.
+   *   binary. Defaults to `binary`.</string>
```

Comment:
all these closing tags are incorrect - but I think what we should do is get rid of the `<string>` / `<number>` part of the docs entirely.

Reply:
Ok I will omit these invalid tags